src/validTags.ts

```diff
@@ -0,0 +1,46 @@
+export const validTags = [
+  // The token is used within the Ampleforth ecosystem
+  "ampleforth-ecosystem",
+
+  // The token is used within the Avalanche ecosystem
+  "avalanche",
+
+  // The token is used within the Bitcoin ecosystem
+  "bitcoin",
+
+  // The token is used within the Ethereum ecosystem
+  "ethereum",
+
+  // The token is the network token for a chain
+  "network-token",
+
+  // The token is derived from the network token for a chain
+  "network-token-derived",
+
+  // The token is rebasing
+  "rebases",
+
+  // The token is a staked Ether token
+  "staked-ether",
+
+  // The token is a fiat-backed stablecoin
+  "usd",
+
+  // The token is specifically wrapped Avalanche
+  "wrapped-avalanche",
+
+  // The token uses the Buttonwood Button wrapper
+  "wrapped-button",
+
+  // The token is specifically wrapped Ether
+  "wrapped-ether",
+
+  // The token is a wrapped network token
+  "wrapped-network-token",
+
+  // The token uses the Buttonwood Tranche wrapper
+  "wrapped-tranche",
```

Comment:
This implies that it's wrapping a tranche-token. All the other names are similar: "wrapped-ether" is wrapping ether and "wrapped-bitcoin" is wrapping bitcoin. 
Rename to "tranche-wrapped" or "tranche-token"